index.js

```diff
@@ -66,7 +66,7 @@ function handler(opts) {
 
 			if (!res.headersSent) {
```

Comment:
This will give an JS exception if `res` is undefined... Right? 🙂

Reply:
It is express middleware - there should always be a response if there is a request.